### DISCUSSION TITLE
text-right on buttons

### DISCUSSION THREAD

**Discussion by CologneX** [svelteuidev/svelteui#398] [Q&A] [answered]
Hi, i'm trying to align the text on the button to the start/right side, how can i do that with/without tailwindcss?
I already tried
`<Button variant="subtle" compact ripple class="text-start">
                    Profile
                </Button>`
                
and
`<Button variant="subtle" compact ripple align="right">
                    Wishlist
                </Button>`
                

This is the full code:
```
<div class="flex-grow grid-flow-row grid">
                <Button variant="subtle" compact ripple>
                    Profile
                </Button>
                <Button variant="subtle" compact ripple align="right">
                    Wishlist
                </Button>
                <Button variant="subtle" compact ripple class="text-start" >
                    Transaction
                </Button>
                <Button variant="subtle" compact ripple class="text-end" >
                    Cart
                </Button>
</div>
```

I also tried <Group> and still got the same result
Thank you

**Comment by BeeMargarida** [comment, marked as answer]
Something like this would work, for example by setting the text to the left. Button uses flex inside to position the text and icons.

```
<Button compact fullSize override={{ justifyContent: 'flex-start'}}>
        Profile
</Button>
```

**Reply by CologneX** [reply]
Thank you!
